client/src/Root.js

```diff
@@ -30,7 +30,7 @@ class UrlSet extends Component{
               <Route path="/student/projects/:project_id/groups/:group_id" exact strict component={StudentGroup}/>  
               <Route path="/admin/home" exact strict component={AdminMainMenu}/>  
               <Route path="/admin/notifications" exact strict component={AdminNotification}/> 
-              <Route path="/admin/projects/new" exact strict component={AdminCreateProject}/> 
+              <Route path="/admin/projects/create/new" exact strict component={AdminCreateProject}/> 
```

Comment:
this is not a restful route https://restfulapi.net/resource-naming/

Reply:
I know, but the question is that since we set the project route : /admin/projects/:project_id
the previous route: /admin/projects/new   many make project_id = new, which may cause problems.
So how to fix it?